hudi-cli/src/test/java/org/apache/hudi/cli/commands/TestCleansCommand.java

```diff
@@ -56,8 +58,16 @@ public void init() throws IOException {
 
     String tableName = "test_table";
     tablePath = basePath + File.separator + tableName;
-    propsFilePath = this.getClass().getClassLoader().getResource("clean.properties").getPath();
-
+    propsFilePath = TestCleansCommand.class.getClassLoader().getResource("clean.properties").getPath();
+    if (propsFilePath == null) {
+      System.out.println("-------------------");
```

Comment:
Please make sure we do not use STDOUT in the test case.

Reply:
> Please make sure we do not use STDOUT in the test case.

Yes, just for check something. There are some different between local and CI environment. Will remove them late.